README.md

```diff
@@ -58,19 +58,21 @@ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 1. Install XCode and dependencies
 ```
 xcode-select --install
-brew install protobuf openssl flex bison icu4c pkg-config
+brew install protobuf openssl flex bison icu4c pkg-config m4
```

Comment:
We build all extensions in the docker image so it's fine if the developers don't install the dependencies in the host machine

Reply:
But the problem is that I couldn't run `make` locally without installing `m4` on my Mac... Not sure I understand what you're saying...

Reply:
Shouldn't this be a separate PR? Is it specific to this PR?

Reply:
https://github.com/neondatabase/neon/pull/9216